### PR TITLE
[en] Update networking model link

### PR DESCRIPTION
**Pull request by KoditkarVedant**
As we have renamed the master branch to main. There are few links which still refers to the old master branch. GitHub does it's best to redirect those links to main branch this is additional redirection besides all other links we have across the document are pointed to main so it make sense to maintain the consistency to update these links.

### CODE REVIEW

File: content/en/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ These repo labels let reviewers filter for PRs and issues by language. For examp
 
 ### Team review 
 
-L10n teams can now review and approve their own PRs. For example, review and approval permissions for English are [assigned in an OWNERS file](https://github.com/kubernetes/website/blob/master/content/en/OWNERS) in the top subfolder for English content. 
+L10n teams can now review and approve their own PRs. For example, review and approval permissions for English are [assigned in an OWNERS file](https://github.com/kubernetes/website/blob/main/content/en/OWNERS) in the top subfolder for English content.
```
Comment (nate-double-u) [inline content/en/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md:44]: Generally we don't update blog posts that are older than a year-we leave them as an archive. In this case, as the master link redirects to the main link automatically (due to GitHub's redirect system), I don't think we should make this change here.

File: content/zh/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md [hunk 2]
```diff
@@ -64,8 +64,8 @@ Prow 根据文件路径自动添加语言标签。感谢 SIG Docs 贡献者 [Jun
 <!-- ### Team review  -->
 ### 团队审核
 
-<!-- L10n teams can now review and approve their own PRs. For example, review and approval permissions for English are [assigned in an OWNERS file](https://github.com/kubernetes/website/blob/master/content/en/OWNERS) in the top subfolder for English content.  -->
-L10n 团队现在可以审查和批准他们自己的 PR。例如，英语的审核和批准权限在位于用于显示英语内容的顶级子文件夹中的 [OWNERS 文件中指定](https://github.com/kubernetes/website/blob/master/content/en/OWNERS)。
+<!-- L10n teams can now review and approve their own PRs. For example, review and approval permissions for English are [assigned in an OWNERS file](https://github.com/kubernetes/website/blob/main/content/en/OWNERS) in the top subfolder for English content.  -->
+L10n 团队现在可以审查和批准他们自己的 PR。例如，英语的审核和批准权限在位于用于显示英语内容的顶级子文件夹中的 [OWNERS 文件中指定](https://github.com/kubernetes/website/blob/main/content/en/OWNERS)。
```
Comment (nate-double-u) [inline content/zh/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md:68]: As above, I'd recommend not updating blog entries that are older than one year.

**Review by nate-double-u — COMMENTED** [review]
Hi @KoditkarVedant, thanks for this!

I've made some inline comments regarding old blog posts:

File: content/zh/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md (hunk 2 above)
Comment (KoditkarVedant) [reply content/zh/blog/_posts/2018-11-08-kubernetes-docs-update-i18n.md:68]: I've reverted the changes.

**Comment by KoditkarVedant** [comment]
@nate-double-u Thank you for the review. I've resolved the comments.

File: content/ko/docs/concepts/cluster-administration/networking.md [hunk 3]
```diff
@@ -118,7 +118,7 @@ Azure CNI는 [Azure 쿠버네티스 서비스(Azure Kubernetes Service, AKS)](ht
...
+[CNI-Genie](https://github.com/Huawei-PaaS/CNI-Genie)는 쿠버네티스가 런타임 시 [쿠버네티스 네트워크 모델](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/cluster-administration/networking.md#the-kubernetes-network-model)의 [서로 다른 구현에 동시에 접근](https://github.com/Huawei-PaaS/CNI-Genie/blob/master/docs/multiple-cni-plugins/README.md#what-cni-genie-feature-1-multiple-cni-plugins-enables)할 수 있는 CNI 플러그인이다. 여기에는 [플라넬(Flannel)](https://github.com/coreos/flannel#flannel), [캘리코](https://docs.projectcalico.org/), [로마나(Romana)](https://romana.io), [위브넷(Weave-net)](https://www.weave.works/products/weave-net/)과 같은 [CNI 플러그인](https://github.com/containernetworking/cni#3rd-party-plugins)으로 실행되는 모든 구현이 포함된다.
```
Comment (nate-double-u) [inline content/ko/docs/concepts/cluster-administration/networking.md:121]: I think this page has changed a bit, and the link here should be: `https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/cluster-administration/networking.md#how-to-implement-the-kubernetes-networking-model`

Comment (jihoon-seo) [reply content/ko/docs/concepts/cluster-administration/networking.md:121]: How about this (for Korean page)? 😊
(CC @kubernetes/sig-docs-ko-reviews)

```suggestion
[CNI-Genie](https://github.com/Huawei-PaaS/CNI-Genie)는 쿠버네티스가 런타임 시 [쿠버네티스 네트워크 모델](/ko/docs/concepts/cluster-administration/networking/#쿠버네티스-네트워크-모델)의 [서로 다른 구현에 동시에 접근](https://github.com/Huawei-PaaS/CNI-Genie/blob/master/docs/multiple-cni-plugins/README.md#what-cni-genie-feature-1-multiple-cni-plugins-enables)할 수 있는 CNI 플러그인이다. 여기에는 [플라넬(Flannel)](https://github.com/coreos/flannel#flannel), [캘리코](https://docs.projectcalico.org/), [로마나(Romana)](https://romana.io), [위브넷(Weave-net)](https://www.weave.works/products/weave-net/)과 같은 [CNI 플러그인](https://github.com/containernetworking/cni#3rd-party-plugins)으로 실행되는 모든 구현이 포함된다.
```

Comment (KoditkarVedant) [reply content/ko/docs/concepts/cluster-administration/networking.md:121]: @nate-double-u I believe what @jihoon-seo mentioned is what we may need to update in Korean document as the heading in Korean still reads "Kubernetes Network Model". 

What you are saying is the case with English document where I see the heading has changed but the link is not updated with latest fragment.

Let me know your thoughts. I will make changes accordingly.

Comment (nate-double-u) [reply content/ko/docs/concepts/cluster-administration/networking.md:121]: This seems better to me @jihoon-seo 😊

Comment (KoditkarVedant) [reply content/ko/docs/concepts/cluster-administration/networking.md]: @nate-double-u updated the docs with correct links.

**Comment by nate-double-u** [comment]
/lgtm
I've followed all the links changed and they look correct.

**Comment by jihoon-seo** [comment]
/approve
as one of Korean approvers.

/lgtm

**Comment by KoditkarVedant** [comment]
@nate-double-u @jihoon-seo @bells17 @ClaudiaJKang Friendly ping. I've rebase with master. Need your help to get this PR across the line.

**Comment by tengqm** [comment]
Please remove zh localization pages from the commits. We need to examine whether each page change is a full sync of the English page.

**Comment by KoditkarVedant** [comment]
@tengqm I'm okay with reverting the zh files but the actual change set is not regarding the translation but the more of hyperlink which is pointing to same file in master branch. I just wanted it to consistently refer branch main instead of master.

**Comment by KoditkarVedant** [comment]
@tengqm as asked I've reverted changes in zh localization pages.

**Comment by nate-double-u** [comment]
I've rechecked the URL that changed since my last look, and it's working. Note, there still looks like there is a zh file changed (it's a small change, just whitespace).

/lgtm

**Comment by reylejano** [comment]
@KoditkarVedant , once a change is made to the English site then it can be changed in localizations. Typically a PR proposes a change to one localization.
Please remove all other changes to localizations from this PR other than English.
Once the change is made to the English site then open PRs to change localizations.

**Comment by KoditkarVedant** [comment]
@reylejano I've only kept changes to English localization files. I will create PR per localization as you suggested.

**Comment by jihoon-seo** [comment]
/remove-language de
/remove-language fr
/remove-language id
/remove-language ja
/remove-language ko
/remove-language ru
/remove-language zh

- Please squash the commits into single commit. 😊
- In this PR, there is no change that 'updates hyperlinks to point to main branch',
  so it seems that the title of this PR needs update. 😊

**Comment by kbhawkey** [comment]
Thanks @KoditkarVedant . The changes look great.
/lgtm

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 4]
```diff
@@ -79,7 +79,7 @@ addressing, and it can be used in combination with other CNI plugins.
...
+[CNI-Genie](https://github.com/cni-genie/CNI-Genie) is a CNI plugin that enables Kubernetes to [simultaneously have access to different implementations](https://github.com/cni-genie/CNI-Genie/blob/master/docs/multiple-cni-plugins/README.md#what-cni-genie-feature-1-multiple-cni-plugins-enables) of the [Kubernetes network model](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model) in runtime. This includes any implementation that runs as a [CNI plugin](https://github.com/containernetworking/cni#3rd-party-plugins), such as [Flannel](https://github.com/flannel-io/flannel#flannel), [Calico](https://projectcalico.docs.tigera.io/about/about-calico/), [Weave-net](https://www.weave.works/oss/net/).
```
Comment (kbhawkey) [inline content/en/docs/concepts/cluster-administration/networking.md:82]: nit: This link refers back to an in page heading,
```
[Kubernetes network model](#how-to-implement-the-kubernetes-networking-model)
```

**Comment by kbhawkey** [comment]
Thanks @KoditkarVedant . Changes look good.
/approve

**Pull request merged**